src/FffGcodeWriter.cpp

```diff
@@ -106,8 +106,9 @@ void FffGcodeWriter::writeGCode(SliceDataStorage& storage, TimeKeeper& time_keep
     }
 
     size_t total_layers = 0;
-    for (SliceMeshStorage& mesh : storage.meshes)
+    for (auto& mesh_ptr : storage.meshes)
```

Comment:
Good practices with the `auto` keyword indicate that you should use it only when the type is clearly explicited somehow around it, which does not seem to be the case here. What do you think ?

Reply:
I broadly agree with that statement (though I tend to use `auto` in `for`-loops as well, as the type can be looked up in the collection -- especially if it's a 'longer' type).

In any case, I'll change this to not use `auto`.

Reply:
Resolved in 74456d083079cc4025c7d7d68507d6a0152e97f6